didel/courses.py

```diff
@@ -116,12 +120,20 @@ def populate(self, soup, session):
         header = soup.select('.courseInfos')[0]
         self.title = header.select('h2 a')[0].get_text()
         self.teacher = header.select('p')[0].get_text().split('\n')[-1].strip()
-
         about = soup.select('#portletAbout')
         if about:
             self.about = about[0].get_text().strip()
 
 
+    def docs_and_links(self, path):
+        """
+        synchronize didel documents with user
+        """
+        if not (os.path.isdir(path)):
+            raise IOError
```

Comment:
Add some informations here, we won’t be able to get the cause when catching the exception. Also, why is this `path` variable not used in the rest of the function? Is it really useful?